sdk/src/client/message_interface/response.rs

```diff
@@ -94,6 +95,12 @@ pub enum Response {
     /// - [`SignatureUnlock`](crate::message_interface::Message::SignatureUnlock)
     SignatureUnlock(UnlockDto),
     /// Response for:
+    /// - [`SignEd25519`](crate::message_interface::Message::SignEd25519)
+    Ed25519Signature(Ed25519SignatureDto),
+    /// Response for:
+    /// - [`VerifyEd25519Signature`](crate::message_interface::Message::VerifyEd25519Signature)
```

Comment:
Random thought, I've been trying to figure out which verb is best here, Validate vs Verify. It slightly triggers me that the response for Verify is Valid

Reply:
Yeah asked myself what to use, we have both in use at different places